src/Sarif/Writers/SarifLogger.cs

```diff
@@ -101,11 +100,6 @@ public class SarifLogger : BaseLogger, IDisposable, IAnalysisLogger
             RuleToIndexMap = new Dictionary<ReportingDescriptor, int>(ReportingDescriptor.ValueComparer);
             ExtensionGuidToIndexMap = new Dictionary<Guid, int>();
 
-            if (dataToInsert.HasFlag(OptionallyEmittedData.Hashes))
```

Comment:
>dataToInsert [](http://example.com/codeflow?start=16&length=12)

I'm deleting this aggressive hashing of targets data on initialization for now. First, we previously had at least three mechanisms for hashing, this one, the hash delegate and on-demand hashing via the insert data visitor. This particular mechanism always held all this data in memory, as opposed to the file regions cache mechanism, which only holds that previous 100 files hashed...